src/main/java/study/validator/StringSumValidator.java

```diff
@@ -0,0 +1,33 @@
+package study.validator;
+
+public class StringSumValidator {
+
+    private static final int STANDARD = 0;
+    private static final String NEGATIVE_NUMBER_MESSAGE = "음수는 입력할 수 없습니다.";
+    private static final String NOT_NUMBER_MESSAGE = "숫자가 아닌 값이 입력되었습니다.";
+
+    public static boolean validatePositive(int number) {
+
+        if (number < STANDARD) {
+            throw new IllegalArgumentException(NEGATIVE_NUMBER_MESSAGE);
+        }
+        return true;
+    }
+
+    public static boolean validateBlank(String data) {
+
+        if (data == null || data.trim().isEmpty()) {
+            return false;
+        }
+        return true;
```

Comment:
```suggestion
        return data != null && !data.trim().isEmpty();
```

크게 문제가 되지않는한 간단한 표현이 더 좋다고 생각해요 😄

Reply:
좋은 리뷰 감사합니다! 다음부턴 간단한 로직은 위와 같이 적용해볼게요